style.css

```diff
@@ -0,0 +1,51 @@
+@media screen and (max-width: 1024px) {
```

Comment:
You can consider not bringing the media query since you only work with only mobile version. 👍 👍